urllib3/response.py

```diff
@@ -182,6 +192,38 @@ def tell(self):
         """
         return self._fp_bytes_read
 
+    def _init_length(self, request_method):
+        """
+        Set initial length value for Response content.
+
+        Logic extracted from httplib.py in the standard Python library.
+        """
+        length = self.headers.get('content-length')
+        if length is not None and not self.chunked:
+            try:
+                length = int(length)
```

Comment:
Yes, rejecting would be raising an exception. =)

As to that approach, no, not quite. That *also* violates RFC 7230 which only wants us to accept the header if the entire list is identical. That is, if:

```python
lengths = {val.strip() for val in header_val.split(',')}
if len(lengths) > 1:
    raise StupidHeaderBlockError("What are you thinking?")
length = int(lengths[0])
```